test/lib/skunk/cli/options/argv_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+require "skunk/cli/options/argv"
+
+describe Skunk::Cli::Options::Argv do
+  describe "#output_filename" do
+    context "passing --out=FILE options" do
+      let(:argv) { ["--out=file.txt"] }
+
+      it "parses passed filename" do
+        parser = Skunk::Cli::Options::Argv.new(argv)
+        parser.parse
+        _(parser.output_filename).must_equal "file.txt"
+      end
```

Comment:
@manuca Could you add another check to make sure that the content of file.txt is the right content?

Reply:
@etagwerker When I created this test I thought about this but realized the report output seems difficult to make deterministic because of the dependency on git commits for instance. I saw we created and injected a mock status reporter to make the output deterministic in `status_reporter_test.rb` but I don't see this possible at this level of testing.

Do you have ideas on how it would be best to test the output at this level?

Reply:
@manuca you can probably get around the non-deterministic problem with git commits by stubbing the method that calculates the churn of the file.